### ISSUE TITLE
COLLADA export issue

### ISSUE THREAD

**Issue by DanMurtin** [3dcitydb/importer-exporter#76]
Hello,
I am having issues with COLLADA export and viewing in citydb_web_map. I import the files, manage to export and view geometry file in citydb_web_map, but when it comes to Collada, i stumble upon an issue. At first, I had no themes at all (rgbTexture), but  now I managed to get FMETheme (i used FME to export GML file), but when i open it in citydb_web_map, it shows the data, but it is all black. 

![untitled](https://user-images.githubusercontent.com/46754222/51472291-58d34580-1d79-11e9-9091-a012b859b6a6.png)

And here is the collada file opened in FME Inspector:
![sd](https://user-images.githubusercontent.com/46754222/51472864-0135d980-1d7b-11e9-847b-e1b0283c2586.png)

Here is the object I am showing:
[object.zip](https://github.com/3dcitydb/importer-exporter/files/2778701/object.zip)

Thank you.

**Comment by Son-HNguyen** [comment]
Hi,

apologies for the late reply. I see you have improved the data set since the last email. 👍 
Using the latest version of the Importer/Exporter and the 3DCityDB Web Map Client, I managed to export and visualize the glTF files:
![image](https://user-images.githubusercontent.com/5765503/51530551-7ff24b80-1e3b-11e9-8c86-675c01432997.png)

For reference, here are the COLLADA/KML/glTF settings that I used:
![image](https://user-images.githubusercontent.com/5765503/51530716-e0818880-1e3b-11e9-84bb-d2959ba10489.png)

Is there any error in the browser's developer mode? Are the web client and the glTF files hosted on the same domain/machine?
